.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     'jest/valid-title': ['error', { ignoreTypeOfDescribeName: true }],
     'no-console': ['error', { allow: [''] }],
     '@jambit/typed-redux-saga/delegate-effects': 'error',
+    'jest/no-disabled-tests': 'off',
```

Comment:
I'd prefer if we didn't do that globally. Otherwise it's easy to disable a test while testing and then actually merging it.
Could we use`// eslint-disable-next-line jest/no-disabled-tests` where needed instead?